### ISSUE TITLE
posible code error on /core/classes/fieldset.php

### ISSUE THREAD

**Issue by monsonis** [fuel/core#63]
/core/classes/fieldset.php on line 126:

```
    if (isset($config['form_instance']) && $config['form_instance'] instanceof Form)
    {
        $this->validation = $config['form_instance'];
    }
```

this maybe should be:

```
    if (isset($config['form_instance']) && $config['form_instance'] instanceof Form)
    {
        $this->form = $config['form_instance'];
    }
```

**Comment by kbanman** [comment]
Pull request here: https://github.com/fuel/core/pull/67

**Comment by jschreuder** [comment]
fixed by fuel/core@95da8fe
